source/grammar/qasm3Parser.g4

```diff
@@ -220,11 +220,17 @@ argumentDefinition:
     | (CREG | QREG) Identifier designator?
     | arrayReferenceType Identifier
 ;
+gateParameter:
```

Comment:
Name bikeshedding: to people prefer if this is called `gateArgument` instead?

Reply:
I don't have a strong preference. I think naively I've always thought of the parameters as those parts of the definition that determine the unitary operation, up to isomorphism, and then the arguments as specifying what qubit subspace this applies to.

With that said, there is a tiny bit of precedent for either, namely
1. In the Python AST, `QuantumGateDefinition` and `QuantumGate` use the word `argument` instead of `parameter`
2. In the spec, e.g. https://openqasm.com/language/gates.html#parameterized-gates, we tend to use the word _parameter_

Reply:
Right, we seem to use "parameter" for this in the spec, but "argument" in the Python AST.

Reply:
This grammar file also seems to use `argument`. E.g. `externArgumentList`. Should we change it?